src/index.js

```diff
@@ -1,7 +1,114 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import L from 'leaflet';
+ReactDOM.render(	
+  <h1>Hello World!</h1>,	
+ document.getElementById('root')
+) 
 
-ReactDOM.render(
-  <h1>Hello World!</h1>,
-  document.getElementById('root')
-)
\ No newline at end of file
+var fs = require('fs');
+var http = require('http');
+var mime = require('mime-types');
+
+var port = process.env.PORT || 5000;
+http.createServer(function (request, response) {
```

Comment:
This function isn't needed to create your express server. A combination of app.get and app.listen should be used to create a generic Express server to work with. 

If the use of http.createServer was intentional, provide documentation as to why that is.